configs/relengworker-prod/config.yml

```diff
@@ -121,8 +121,8 @@ worker_types:
     autoscale:
       algorithm: slo
       args:
-        max_replicas: 15
-        avg_task_duration: 60
+        max_replicas: 60
+        avg_task_duration: 8
```

Comment:
Does this take into account the average run task for the balrogscript or the task itself? (including scriptworker before and after calling `balrogscript`)? Because if former, I think we should probably double this, to have some room there.

Reply:
My queries account for the entirety of task runtime and CoT, so I _think_ this is okay? It doesn't account for time to spin up an instance and run scriptworker, but AFAICT we shouldn't worry about his here. What do you think?

Reply:
Okay, in that case I think we're good, CoT was what I had in mind. Time to spin up instance is not taken into consideration here AFAIK so we should be good.